CHANGELOG.md

```diff
@@ -1,10 +1,17 @@
-# v1.3.5
+# Unpublished
 
 ### `@liveblocks/react`
 
 - Officially mark `useList()`, `useMap()`, and `useObject()` as deprecated in
   JSDoc comments (we stopped recommending them since the release of 0.18)
 
+# v1.3.5
+
+### `@liveblocks/react`
+
+- Deduplicate Comments requests and improve how race conditions are handled
+  during mutations.
+
```

Comment:
v1.3.5 hasn't been published yet (although it's on `main` already). Sorry for the confusion!

```suggestion
# v1.3.5

### `@liveblocks/react`

- Officially mark `useList()`, `useMap()`, and `useObject()` as deprecated in
  JSDoc comments (we stopped recommending them since the release of 0.18)

### `@liveblocks/react`

- Deduplicate Comments requests and improve how race conditions are handled
  during mutations.

```